contracts/bridge/BridgeOperator.sol

```diff
@@ -101,15 +107,35 @@ contract BridgeOperator is Ownable {
     external
     onlyOwner
     {
+        if (operators[_operator]) {
+            return;
```

Comment:
Doesn't it need to be reverted?
require(!operators[_operator])

Reply:
I also curious about this. :)